### ISSUE TITLE
Make DAR robust against missing files

### ISSUE THREAD

**Issue by oliver----** [substance/texture#499]
In Stencila, @apawlik found a case where a previously opened DAR could not be opened anymore.
One source of the error is, that `StencilaArchive/TextureArchive` crashed because of a missing `sheet.xml`

We should make sure, that the DAR can still be opened, and probably inform the user about the problem.

**Comment by michael** [comment]
Should I take this now?

**Comment by oliver----** [comment]
Yeah, maybe investigate how to do it appropriately. In the workaround I have just 'dropped' the unresolvable sessions. Dunno if that has a negative side-effect else where. At least the manuscript should be fixed as well. And there should be a message shown to the user.

**Comment by michael** [comment]
Dropping sessions is ok, though we should also fix the manifest document to not include the dropped entries. Otherwise they'll show up for instance in the tabs, but have no session... will check...

Regarding errors, think for now I just do a console.warn, but agree we should introduce an error API for the loading process.

**Comment by michael** [comment]
@oliver please review #511

**Comment by michael** [comment]
Fixed via https://github.com/substance/texture/pull/511.
